snc.sh

```diff
@@ -12,10 +12,17 @@ XMLLINT=${XMLLINT:-xmllint}
 YQ=${YQ:-yq}
 CRC_VM_NAME=${CRC_VM_NAME:-crc}
 BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
-MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/clients/ocp}
+ARCH=$(uname -m)
 CRC_PV_DIR="/mnt/pv-data"
 SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_rsa_crc"
 
+# Point to proper directory on mirror based on architecture
+if [ "${ARCH}" == "ppc64le" ] || [ "${ARCH}" == "s390x" ]; then
+    MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
```

Comment:
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ also exists, so I would unconditionnally use `https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp`

Reply:
Looks like ppc64le arc is not upto date, if you check x86_64 you can find the 4.5 bits but not present for ppc64 side :(

Reply:
yeah i think for 4.4. and 4.5 ppc64le/s390x will GA later than x86 and 4.6 onward they should release at the same time.